apps/rush-lib/src/api/PackageJsonEditor.ts

```diff
@@ -141,6 +146,19 @@ export class PackageJsonEditor {
         );
       });
 
+      Object.keys(data.resolutions || {}).forEach((packageName: string) => {
+        this._resolutions.set(
+          packageName,
+          new PackageJsonDependency(
+            packageName,
+            devDependencies[packageName],
```

Comment:
Needs to reference the original `data.resolutions` object

Reply:
Good catch!  Fixed